std_msgs/Header header

float32 angular_velocity
float32 steer_angle
float32 drive_torque
float32 brake_torque
float32 reaction_torque

geometry_msgs/Point tire_force
geometry_msgs/Point tire_slip

bool can_slip

float32 combined_tire_slip
float32 suspension_compression
float32 down_force
float32 contact_angle
float32 contact_depth
float32 contact_speed

bool is_grounded

geometry_msgs/Vector3 wheel_velocity